Bootstrapping Prefect builds

schuylermartin45 opened the pull request:
Empty commit to get the Prefect builds running. Build seems to pass on local without deltas.

schuylermartin45 committed:
Bootstrapping Prefect builds

anaconda-pkg-build commented:
<details><summary>Linter check found the following problems:</summary>
The following problems have been found:

ERROR: /tmp/abs_6c_i3o0m95/clone/recipe/meta.yaml:24: host_section_needs_exact_pinnings: Packages in host must have exact version pinnings, except python build tools.
ERROR: /tmp/abs_6c_i3o0m95/clone/recipe/meta.yaml:29: missing_tests: No tests were found.
ERROR: /tmp/abs_6c_i3o0m95/clone/recipe/meta.yaml:36: missing_dev_url: The recipe is missing a dev_url
ERROR: /tmp/abs_6c_i3o0m95/clone/recipe/meta.yaml:36: missing_documentation: The recipe is missing a doc_url or doc_source_url
WARNING: /tmp/abs_6c_i3o0m95/clone/recipe/meta.yaml:36: missing_description: The recipe is missing a description
Errors were found

</details>

schuylermartin45 commented:
....and of course I forgot to check the linter.

schuylermartin45 committed:
[PKG-1834] Minor linter fixes

**Changes**
- This addresses issues found by the linter

**Issues Encountered**
- N/A

cbouss commented on recipe/meta.yaml:
```diff
@@ -34,7 +34,21 @@ about:
   license_family: GPL
```
```suggestion
  license_family: LGPL
```

cbouss replied:
And license is LGPL-2.1-or-later
https://git.gnunet.org/libmicrohttpd.git/tree/COPYING?h=v0.9.76#n7

schuylermartin45 replied:
Damn, I didn't think to check that. I guess that's wrong in conda-forge as well then...
 
Is that something the linter could check for? i.e. Vet what we have against the `LICENSE` file found from the repo supplied by`dev_url`

cbouss replied:
It could, but it doesn't download upstream code at the moment.

schuylermartin45 committed:
Update recipe/meta.yaml

Co-authored-by: Charles Bousseau <16641587+cbouss@users.noreply.github.com>

AndrewVallette reviewed (approved):
LGTM
